core/src/test/scala/org/broadinstitute/dsde/rawls/entities/datarepo/DataRepoEntityProviderSpecSupport.scala

```diff
@@ -88,7 +88,7 @@ trait DataRepoEntityProviderSpecSupport {
   }
 
   val defaultTables: List[TableModel] = List(
-    new TableModel().name("table1").primaryKey(null).rowCount(0)
+    new TableModel().name("table1").primaryKey(null).rowCount(10)
```

Comment:
update the test-fixture data to have a realistic row count; other tests needed updating because of this change.